Civility Optics/Views/Onboarding/ForgotPasswordView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  VerifyView.swift
+//  Civility Optics
+//
+//  Created by Alexander Goebel on 11/11/22.
+//
+
+import SwiftUI
+
+struct ForgotPasswordView: View {
+    @ObservedObject var model = RegisterViewModel()
+    @State var didVerify: Bool?
+    @State var forgotEmail: String = ""
+    @State var sent: Bool = false
+    @State private var code: String = ""
+    @State private var confirm: String = ""
+    @State private var password: String = ""
+    var body: some View {
+        // Page 1: Enter Email
+        VStack{
+            if sent == false {
+                Group {
+                    let str =  "Please enter email for account with forgotten password:"
+                    Text(str)
+                        .font(.headline)
+                        .multilineTextAlignment(.leading)
+                    ZStack {
+                        RoundedRectangle(cornerRadius: 12)
+                            .foregroundColor(.pale)
+                        HStack {
+                            TextField("Email", text: $forgotEmail)
+                                .foregroundColor(.stone)
+                                .keyboardType(.alphabet)
+                                .autocapitalization(.none)
+                            Spacer()
+                        }
+                        .padding(.horizontal)
+                    }.frame(width: 300, height: 40)
+                    Button {
+                        model.forgotPassword(email:forgotEmail)
+                        self.sent = true
```

Comment:
We may want to return something from the backend to determine if email was actually sent to.